test/e2e/metrics_test.go

```diff
@@ -94,6 +94,23 @@ func (s *MetricsSuite) TestDAGMetrics() {
 		})
 }
 
+func (s *MetricsSuite) TestFailedMetric() {
+	s.Given().
+		Workflow(`@testdata/wf-template-status-failed-conditional-metric.yaml`).
```

Comment:
```suggestion
		Workflow(`@testdata/template-status-failed-conditional-metric.yaml`).
```

the `wf-` in the name is not needed anymore